docs/guides/version-1-upgrade.md

```diff
@@ -0,0 +1,369 @@
+---
+layout: ""
+page_title: "Version 1 Upgrade Guide (from version 0)"
+description: |-
+  Version 1.0.0 of the PingFederate Terraform provider is a major release that introduces breaking changes to existing HCL.  This guide describes the changes that are required to upgrade v0.* PingFederate Terraform provider releases to v1.0.0 onwards.
+---
+
+# PingFederate Terraform Provider Version 1 Upgrade Guide (from version 0)
+
+Version `1.0` of the PingFederate Terraform provider is a major release that introduces breaking changes to existing HCL. This guide describes the changes that are required to upgrade `v0.*` PingFederate Terraform provider releases to `v1.*`.
+
+## Why have schemas changed?
+
+As part of ensuring the ongoing maintainability of the Terraform provider integration and to solve functional issues, some resource schemas have changed going to version `1` from version `0`.
+
+The schemas may have changed in the following ways in this release:
+
+* Removal of previously deprecated fields
+* Removal of previously deprecated resources/data sources
+
+The following sections detail the rationale for the above changes, and whether the changes are routine for a major version upgrade or one off changes that aren't expected in future major version changes.
+
+### Removal of previously deprecated fields
+
+Removal of deprecated fields are expected on each major release going forward.  Ping maintains a deprecation and release strategy according to [Terraform provider creation best practices](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning) and support of PingFederate product versions aligns with the [Ping Identity End of Life Policy](https://www.pingidentity.com/en/legal/end-of-life-policy.html).
+
+### Removal of previously deprecated resources/data sources
+
+Removal of deprecated resources / data sources are expected on each major release going forward.  Ping maintains a deprecation and release strategy according to [Terraform provider creation best practices](https://developer.hashicorp.com/terraform/plugin/best-practices/versioning) and support of PingFederate product versions aligns with the [Ping Identity End of Life Policy](https://www.pingidentity.com/en/legal/end-of-life-policy.html).
+
+### Removal of support for previous PingFederate versions
+
+The support of PingFederate product versions aligns with the [Ping Identity End of Life Policy](https://www.pingidentity.com/en/legal/end-of-life-policy.html).  In this major release, supported PingFederate versions remain unchanged, versions `11.2` through `12.1` of PingFederate are supported.
```

Comment:
@henryrecker-pingidentity just checking here that 11.2 is not EOL and that 12.1 is the latest we'll support at the point of `v1.0` release